recipes/mpfr/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
Yes, in gmp build also we saw homebrew installed gmp interfering with conda version